src/Microsoft.DotNet.XHarness.iOS.Shared/TestReporter.cs

```diff
@@ -617,7 +625,15 @@ public async Task<(TestExecutingResult ExecutingResult, string? ResultMessage)>
 
         if (_timedout)
         {
-            result.ExecutingResult = TestExecutingResult.TimedOut;
+            // Did the test execution start?
+            if (_listener.ConnectedTask.IsCompleted && _listener.ConnectedTask.Result)
```

Comment:
This is the same condition as above, wouldn't it be more readable to abstract this to some function with a descriptive name?

Reply:
Good idea, will do